picard/ui/logview.py

```diff
@@ -181,10 +199,18 @@ def __init__(self, parent=None):
         self.hbox.addWidget(self.verbosity_menu_button)
 
         self.verbosity_menu = VerbosityMenu()
-        self._set_verbosity(self.verbosity)
         self.verbosity_menu.verbosity_changed.connect(self._verbosity_changed)
         self.verbosity_menu_button.setMenu(self.verbosity_menu)
 
+        self.debug_opts_menu_button = QtWidgets.QPushButton(_("Debug Options"))
+        self.debug_opts_menu_button.setAccessibleName(_("Debug Options"))
```

Comment:
Would it make sense to create a temporary variable here to avoid having to translate the same string twice?  I ask because I'm still a bit fuzzy on how the i18n stuff works.

Reply:
It would make sense, but it also hides the source a bit to translators.
In this case, that's definitively not an issue, in a loop or in critical performance code caching would make sense.